src/main/java/uk/ac/ebi/eva/pipeline/configuration/io/readers/AccessionReportReaderConfiguration.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.configuration.io.readers;
+
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import org.springframework.batch.item.ItemStreamReader;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.ac.ebi.eva.commons.models.data.Variant;
+import uk.ac.ebi.eva.pipeline.io.readers.AccessionReportReader;
+import uk.ac.ebi.eva.pipeline.parameters.AccessionReportParameters;
+
+import java.io.File;
+import java.io.IOException;
+
+import static uk.ac.ebi.eva.pipeline.configuration.BeanNames.ACCESSION_REPORT_READER;
+
+@Configuration
+public class AccessionReportReaderConfiguration {
+
+    @Bean(ACCESSION_REPORT_READER)
+    @StepScope
+    public ItemStreamReader<Variant> accessionReportReader(AccessionReportParameters accessionReportParameters) throws IOException {
```

Comment:
Any rationale as to why this needs to be a separate parameter class as opposed to the regular InputParameters class?

Reply:
No, deleted and put the parameter in InputParameter class